flink-runtime/src/test/java/org/apache/flink/runtime/executiongraph/ExecutionGraphRestartTest.java

```diff
@@ -595,9 +594,6 @@ public void slotPoolExecutionGraph_ConcurrentSchedulingAndAllocationFailure_Shou
 
 	@Test
 	public void testRestartWithEagerSchedulingAndSlotSharing() throws Exception {
-		// this test is inconclusive if not used with a proper multi-threaded executor
```

Comment:
The executor(futureExecutor) is not used for scheduling/failover anymore since '[FLINK-11417] Make access to ExecutionGraph single threaded from JobMaster main thread' is merged.

I think there is no need to specify the executor now and the tests does not rely on that executor to do any thing.
The idea of testRestartWithEagerSchedulingAndSlotSharing is easy that it tests a eager scheduling job can recover from a task failure.
The idea of testRestartWithSlotSharingAndNotEnoughResources is also easy that the job will keep failing due to not enough resource until the restart strategy suppresses the restarts.
So I think we can just remove the assumption and the executor specification. Like in 6d323bf2657f8f82eac28d2f529fa1dbe36fcde6.

Reply:
ok